docs/source/scala/quickstart.md

```diff
@@ -0,0 +1,159 @@
+# Quick Start Guide
+In this section, we will see how we can work with the Raphtory Scala APIs to create and analyse graphs. The foundations that we have build in the introductory chapters are as much applicable to this section therefore we won't be revisiting them here again!
+
+## Installing Dependencies
+To work with a Scala project you would need Scala, Java and SBT as development dependencies. The dependencies that we have tested the project discussed in this section to work with Raphtory are Scala version `2.13.7`, Java version `11.0.11.hs-adpt` and SBT version `1.8.0`.
+
+**Note**: You can manage these dependencies using [SDKMAN](https://sdkman.io/), if you like.
+
+The Raphtory version that we will be discussing in this section is a "pre-release" version `v0.2.0a7`.
+
+## Create Project
+We can create a new project from Raphtory giter8 template like so:
+```
+$ sbt new Raphtory/raphtory.g8
+```
+
+Besides downloading some of the required dependencies and creating a new project from the template, a couple of project details (with some sensible defaults) are asked, as part of above step, which you might want to change if you would like:
+```
+name [raphtory-quickstart]:
+version [0.1.0-SNAPSHOT]:
+scala_version [2.13.7]:
+sbt_version [1.8.0]:
+organization [com.pometry]:
+package [com.pometry]:
+raphtory_version [0.2.0a7]:
+```
+
+## Create/Retrieve Graph
+To start working with a graph, we can either create it with a user-specified graph id (or get it auto-generate) or retrieve an existing graph for a given graph id. We can make use of following APIs:
+
+- To create a new graph 
+
+    ```scala
+    def runWithNewGraph[T](graphID: String = createName, persist: Boolean = false)
+    ```
+    **Note**: The default behaviour is to destory any new graph upon exiting the scope of raphtory context unless chosen to persist explicitly.
```

Comment:
make this an actual note (see above)

Reply:
typo: destory -> destroy

Reply:
the graph is already destroyed after the function finishes, not tied to the scope of the context (which I would interpret to mean that it is still available until the context you called `runWithNewGraph` on goes out of scope.